### DISCUSSION TITLE
怎么调用secretflow中的ecdh-psi接口

### DISCUSSION THREAD

**Discussion by 123abcs** [secretflow/secretflow#1504] [Q&A]
怎么调用secretflow中的ecdh-psi接口

**Comment by 1139763082** [comment]
您参考下这个，有什么问题再提问。https://www.secretflow.org.cn/zh-CN/docs/secretflow/v1.9.0b2/tutorial/PSI_On_SPU#%E6%8E%A5%E4%B8%8B%E6%9D%A5
